apps/lockfile-explorer-web/src/containers/LockfileEntryDetailsView/index.tsx

```diff
@@ -6,18 +6,26 @@ import { ScrollArea, Text } from '@rushstack/rush-themed-ui';
 import styles from './styles.scss';
 import appStyles from '../../App.scss';
 import { IDependencyType, LockfileDependency } from '../../parsing/LockfileDependency';
+import { readPackageJsonAsync } from '../../parsing/getPackageFiles';
 import { useAppDispatch, useAppSelector } from '../../store/hooks';
 import { pushToStack, selectCurrentEntry } from '../../store/slices/entrySlice';
 import { ReactNull } from '../../types/ReactNull';
 import { LockfileEntry } from '../../parsing/LockfileEntry';
 import { logDiagnosticInfo } from '../../helpers/logDiagnosticInfo';
 import { displaySpecChanges } from '../../helpers/displaySpecChanges';
+import { IPackageJson } from '../../types/IPackageJson';
 
 enum DependencyType {
   Determinant,
   TransitiveReferrer
 }
 
+enum DependencyKey {
+  DEP = 'dependencies',
+  DEVDEP = 'devDependencies',
+  PEERDEP = 'peerDependencies'
```

Comment:
Our coding convention is to spell out English words rather than using abbreviations such as "dep".

Compare with this example: https://api.rushstack.io/pages/rush-lib.dependencytype/#enumeration-members